artifactory/commands/npm/rtPublishHandler.go

```diff
@@ -0,0 +1,131 @@
+package npm
+
+import (
+	"errors"
+	"fmt"
+	buildinfo "github.com/jfrog/build-info-go/entities"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	specutils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/content"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+type npmRtUpload struct {
+	*NpmPublishCommand
+}
+
+func (nru *npmRtUpload) upload() (err error) {
+	for _, packedFilePath := range nru.packedFilePaths {
+		log.Debug("Deploying npm package.")
```

Comment:
this log is not giving good debug information, either add more details to log or you can remote it.